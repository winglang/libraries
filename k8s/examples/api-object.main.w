bring ".." as k8s;

new k8s.ApiObject(
  apiVersion: "v1",
  kind: "ConfigMap",
  metadata: {
    name: "your-config",
  },
  spec: {
    data: {
      "key": "value",
    },
  },
);
