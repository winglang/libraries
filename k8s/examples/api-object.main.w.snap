apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    app: bang-bang
    fang: fang
  name: your-config
  namespace: flanging
spec:
  data:
    key: value
